statistics/column.go

```diff
@@ -140,13 +138,10 @@ func (c *Column) MemoryUsage() CacheItemMemoryUsage {
 // Currently, we only load index/pk's Histogram from kv automatically. Columns' are loaded by needs.
 var HistogramNeededItems = neededStatsMap{items: map[model.TableItemID]struct{}{}}
 
-// IsInvalid checks if this column is invalid.
+// ColumnStatsIsInvalid checks if this column is invalid.
 // If this column has histogram but not loaded yet,
 // then we mark it as need histogram.
-func (c *Column) IsInvalid(
-	sctx sessionctx.Context,
-	collPseudo bool,
-) (res bool) {
+func ColumnStatsIsInvalid(colStats *Column, sctx sessionctx.Context, histColl *HistColl, cid int64) (res bool) {
```

Comment:
`colsStats` seems redundant since it's equal to `histColl.Columns[cid]`, can we remove it?